<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Makoto</class>
 <widget class="QWidget" name="Makoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Makoto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: white;
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="page">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="input_username">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
color: black;
outline: none;
border-radius: 10px;
padding: 5px 9px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="header_3">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>170</y>
       <width>801</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NK-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="header_4">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>260</y>
       <width>801</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NK-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="header_2">
     <property name="geometry">
      <rect>
       <x>-21</x>
       <y>30</y>
       <width>801</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NK-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Login to</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_password">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
color: black;
outline: none;
border-radius: 10px;
padding: 5px 9px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="header">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>60</y>
       <width>800</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 46px;a=</string>
     </property>
     <property name="text">
      <string>Makoto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>380</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NK-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
background-color: #4629f2;
color: white;
border: 3px solid #555555;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="header_5">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>60</y>
       <width>800</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 46px;a=</string>
     </property>
     <property name="text">
      <string>Makoto/Hashmap Statistics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_logout">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NK-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
background-color: #ff00d2;
color: white;
border: 3px solid #555555;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QLabel" name="nickname">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Nickname here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="header_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Latest items (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tag2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>freeek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="header_26">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Most frequently used tags (5)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tag5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>430</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>freeek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tag4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>380</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>freeek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tag1">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>freeek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tag3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>UD Digi Kyokasho NP-B</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>freeek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
